### PR TITLE
Highlight clicked menu-item

### PR DESCRIPTION
**Pull request by OhadAsh**
Highlight clicked menu-item

### CODE REVIEW

File: chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.js [hunk 1]
```diff
@@ -30,6 +31,7 @@ const useStyles = makeStyles((theme) => ({
   root: {
     display: "flex"
   },
+  "menu-app-bar": {},
```
Comment (smartcity-hit) [inline chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.js:34]: remove.

File: chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.js [hunk 2]
```diff
@@ -125,8 +131,9 @@ const MenuAppBar = () => {
         className={clsx(classes.appBar, {
           [classes.appBarShift]: openMenuAppBar
         })}
+
       >
-        <Toolbar className={classes.temp}>
+        <Toolbar className={classes.temp + " menu-app-bar"}>
```
Comment (smartcity-hit) [inline chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.js:136]: remove "menu-app-bar"

File: chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.js [hunk 3]
```diff
@@ -174,9 +181,9 @@ const MenuAppBar = () => {
         <Divider />
 
         <Link to="/AdminPanel">
-          <List>
+          <List className="menu-app-bar">
```
Comment (smartcity-hit) [inline chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.js:184]: remove css class.

File: chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.scss [hunk 4]
```diff
@@ -0,0 +1,13 @@
+.menu-app-bar {
+    .nav-link {
+      
+      cursor: pointer;
+      transition: all 0.3s linear;
+      &.active {
+        background-color: #3f51b5;
```
Comment (smartcity-hit) [inline chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.scss:7]: change color to #DFD8D8

File: chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.js [hunk 5]
```diff
@@ -174,9 +181,9 @@ const MenuAppBar = () => {
...
 
-            <ListItem button key={`nav-link ${pathname.includes('AdminPanel') ? 'active' : ''}`}>
+            <ListItem button className={`nav-link ${pathname.includes('AdminPanel') && activeTab === 0 ? 'active' : ''}`} onClick={() => { onClickTab(0) }}>
```
Comment (smartcity-hit) [inline chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.js:186]:
```suggestion
            <ListItem button className={`nav-link ${pathname.includes('AdminPanel') && activeTab === 0 ? 'active' : ''}`} onClick={ onClickTab(0) }>
```

File: chillers-FE-UI/src/Components/MenuAppBar/MenuItems/CountersMenuItem.js [hunk 6]
```diff
@@ -52,8 +57,8 @@ export default function CountersMenuItem(props) {
       </Link>
 
       <Collapse in={open} timeout="auto" unmountOnExit>
-        <List component="div" disablePadding>
-          <ListItem button className={`nav-link ${pathname.includes('CountersDevices') ? 'active' : ''}`}>
+        <List className="menu-app-bar" component="div" disablePadding>
+          <ListItem button className={`nav-link ${pathname.includes('CountersDevices') && activeTab === 1 ? 'active' : ''}`} onClick={() => { onClickTab(1) }}>
```
Comment (smartcity-hit) [inline chillers-FE-UI/src/Components/MenuAppBar/MenuItems/CountersMenuItem.js:61]: why 1?

Comment (OhadAsh) [reply chillers-FE-UI/src/Components/MenuAppBar/MenuItems/CountersMenuItem.js:61]: I’m just mimicking what they did in admin page. From what I understood it saves the number of active tabs in onclick so I left it as is.
It is also implemented that why in chillers. If its not necessary I would like an explanation maybe I didn’t understand something

File: chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.scss (hunk 4 above)
Comment (OhadAsh) [reply chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.scss:7]: Will do

File: chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.js (hunk 1 above)
Comment (OhadAsh) [reply chillers-FE-UI/src/Components/MenuAppBar/MenuAppBar.js:34]: No problem its not necessary forgot to take it down

**Pull request merged**